app/src/main/java/org/mozilla/tv/firefox/IntentValidator.kt

```diff
@@ -17,6 +17,7 @@ import org.mozilla.tv.firefox.telemetry.TelemetryIntegration
 import org.mozilla.tv.firefox.utils.UrlUtils
 
 data class ValidatedIntentData(val url: String, val source: Session.Source)
+private const val ACTIVE_EXPERIMENTS_KEY = "activeExperiments"
```

Comment:
nit: The value should probably be `EXTRA_KEY_ACTIVE_EXPERIMENTS` or `KEY_ACTIVE_EXPERIMENTS`: it makes it more obvious how certain constants are to be used when it's prefixed. I asked Daniela to make this change too.